crates/cargo-util/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo-util"
-version = "0.2.4"
```

Comment:
This might be unnecessary? We've bump to 0.2.4 but yet released it. https://github.com/rust-lang/cargo/commit/2f0ecf54254b448bd24b4493c6686aafaa6be72f

Reply:
Oh, indeed, sorry I somehow got confused.